docs/manual/default-configs/maximal.md

```diff
@@ -1,7 +1,7 @@
 # Maximal {#sec-default-maximal}
 
 ```bash
-$ nix shell github:notashelf/nvf#maximal test.nix
+$ nix run github:notashelf/nvf#maximal test.nix
```

Comment:
```suggestion
$ nix run github:notashelf/nvf#maximal -- test.nix
```

Also missing the dashes to past `test.nix` to the built Neovim instead of Nix. Could you fix this while you're at it?